content/error-handling/index.textile

```diff
@@ -0,0 +1,130 @@
+---
+title: Error handling
+meta_description: "Error handling in Ably supported apps, including troubleshooting techniques, logging options, and tools for debugging."
+---
+
+Errors can occur in various scenarios when using Ably, such as "invalid inputs":/error-handling/error-codes#errors in requests, authentication issues, or connection problems caused by network disruptions. Proper error handling is essential for building a reliable application and troubleshooting.
+
+* Set up a custom log handler to capture and manage errors in a way that suits your requirements.
+* Meta channels allow you to monitor errors that might not otherwise be visible to clients, providing additional insights into issues.
+* The Dev console in your Ably "dashboard":https://ably.com/accounts/any is a quick and easy way to inspect errors and events, especially during development or debugging.
+
+Common errors in Ably-supported apps usually arise from:
+
+- *Authentication issues* := Invalid or expired tokens, incorrect API keys.
+- *Permission errors* := Missing or insufficient capabilities for publishing or subscribing.
+- *Channel attachment failures* := Incorrect channel names, network connectivity problems, or unauthorized access.
+
+h2.(#format) Error format
+
+All errors returned by Ably are standardized and use the @ErrorInfo@ object:
+
+```[text]
+{
+  code: 40005,
+  statusCode: 400,
+  cause: Authentication,
+  nonfatal: false,
+  href: 'https://help.ably.io/error/40005'
+}
+```
+
+The following explains each property of an @ErrorInfo@ object:
+
+- *code* := Ably-specific numeric code indicating the error type.
+- *statusCode* := An HTTP status code providing broader context, such as 400 for a bad request.
+- *cause* := A brief description of the issue.
+- *nonfatal* := A boolean indicating whether the error is critical.
+- *Href* := A direct link to Ably's documentation or for quick troubleshooting references.
+
+h2(#logging). Logging
+
+Most Ably SDKs allow you to customize the function that handles logging. This function is usually set in the options when configuring a client, such as the @ClientOptions@ object for Pub/Sub.
+
+Two separate properties can be set:
+
+- *logHandler* :=  Provide a custom function for each line of log output.
+- *logLevel* := The verbosity of the logging output, from silent to trace. In some SDKs this is numeric, and in others string.
+
+The following example configures the Ably client to log only error messages (level 1) and processes them with the function @logWriteFun()@:
+
+```[text]
+const ablyClient = new AblyClient(ably, {
+  logHandler: logWriteFunc,
+  logLevel: 1 // Errors only
+});
+```
```

Comment:
```suggestion
```[javascript]
const ablyClient = new Ably.Realtime({
  key: {{API_KEY}}, 
  logHandler: logWriteFunc,
  logLevel: 1 // Errors only
});
```
```

I think you need something like this.